src/widgets/StackedTables.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable react/forbid-prop-types */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { StyleSheet, View } from 'react-native';
+
+import MiniTable from './SimpleTable';
+
+import { generateUUID } from '../database';
+
+/**
+ * Simple component which renders SimpleTable components
+ * on top one one another. By default will use 100% of
+ * the width and height of it's parent. To give space
+ * between each table, the tablesHeight prop is the space
+ * all tables should occupy, leaving room between each if
+ * < 100%. i.e : 90% will occupy 90% of the height and use
+ * flexbox space-between, so 10% of the height is split evenly
+ * between each table.
```

Comment:
Not following format of other doc blocks

Reply:
done 58b5855d83a44af0d710c3b6e03effc6f94cb806